starter-code/pizza.js

```diff
@@ -1 +1,33 @@
-// Write your Pizza Builder JavaScript in this file.
+$(document).ready(function() {
+  $(".btn-pepperonni").click(function() {
+    $(".pep").toggle();
+    $("aside ul li:first-child").toggle();
```

Comment:
this works, but semantically speaking, it could be better. I know that the exercise asks not to touch the html, but in real life, it would probably make more sense to add a class name to the price and select it that way :)